Program.cs

```diff
@@ -0,0 +1,405 @@
+            invoices.Sellers = objSellers;
+
+            string optionSelected = "y";
+            int numberRegistry = 0;
+            decimal total = 0;
+
+            invoices.InvoiceDetails = new List<InvoiceDetails>();
+
+            while (optionSelected == "y")
+            {
+                Console.WriteLine("Id Product: ");
+                if (!int.TryParse(Console.ReadLine(), out int idProduct))
+                {
+                    return;
+                }
+
+                Products objProducts = productsList.SingleOrDefault(producto => producto.IdProduct == idProduct);
+                PrintData(objProducts);
+
+                Console.WriteLine("Quantity: ");
+                if (!int.TryParse(Console.ReadLine(), out int quantity))
+                {
+                    return;
+                }
+
+                InvoiceDetails invoiceDetails = new InvoiceDetails();
+                invoiceDetails.IdInvoiceDetail = numberRegistry + 1;
+                invoiceDetails.Products = objProducts;
+                invoiceDetails.Price = objProducts.Price;
+                invoiceDetails.Quantity = quantity;
+
+                total = total + objProducts.Price*quantity;
+
+                invoices.InvoiceDetails.Add(invoiceDetails);
+
+                numberRegistry++;
+
+                optionSelected = ConfirmationAddDetails();
+            }
+
+            invoices.Amount = total;
+            invoicesList.Add(invoices);
+
+            MenuMain();
+            ProcessMain();
+        }
+        private static void ReportSales()
+        {
+            decimal TotalAmount = 0;
+
+            Console.WriteLine("Sales Record");
+
+            invoicesList.ForEach(itemInvoices => {
+                TotalAmount = TotalAmount + itemInvoices.Amount;
+                itemInvoices.InvoiceDetails.ForEach(itemInvoiceDetails =>
+                {
+                        Console.WriteLine($"Id Invoice: {itemInvoices.IdInvoice}  Id Invoice Detalle: {itemInvoiceDetails.IdInvoiceDetail} Product: {itemInvoiceDetails.Products.Description}   Customer: {itemInvoices.Customers.Name} Price: {itemInvoiceDetails.Price}    Quantity: {itemInvoiceDetails.Quantity} Total: {itemInvoiceDetails.Price * itemInvoiceDetails.Quantity}");
+                });
+            });
+
+            Console.WriteLine($"Grand Total: {TotalAmount}");
+
+            MenuMain();
+            ProcessMain();
+        }
+        private static void ReportCustomersAge()
+        {
+            Console.WriteLine("Customers over 55");
+
+            List<Customers> objCustomersList = new List<Customers>();
+            objCustomersList = customersList.Where(c => DateTime.Today.Year - Convert.ToDateTime(c.BirthDate).Year > 55).ToList();
+
+            foreach(Customers customers in objCustomersList)
+            {
+                Console.WriteLine($"Id Customer: {customers.IdCustomer}  Name: {customers.Name}  Birth Date: {customers.BirthDate}");

```

Comment:
if you have a ToString method tha make this you must to use it instead to write again the same code.